renderdoc/data/glsl/mesh.geom

```diff
@@ -22,6 +22,12 @@
  * THE SOFTWARE.
  ******************************************************************************/
 
+#ifdef OPENGL_ES
+//#extension GL_EXT_geometry_shader : enable
+//#extension GL_OES_geometry_shader : enable
+//#extension GL_EXT_geometry_point_size : enable
```

Comment:
That makes sense. If at some point in the future we need different extensions for GL and Vulkan then we can add it, for now `#extension_gles` and `#extension_nongles` sounds good.